its/target-platform-e43/target-platform-its-e43.target

```diff
@@ -17,7 +17,7 @@
 </location>
 <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
 <unit id="org.python.pydev.feature.feature.group" version="0.0.0"/>
-<repository location="http://www.pydev.org/update_sites/5.2.0"/>
+<repository location="https://dl.bintray.com/fabioz/pydev/5.2.0"/>
```

Comment:
Why is this better?

Reply:
It was failing on cix and I don't understand why. Seems related to the redirect + composite update site.